Cabal/Cabal.cabal

```diff
@@ -342,6 +342,14 @@ library
   if !impl(ghc >= 7.0)
     default-extensions: CPP
 
+executable cabal-setup
```

Comment:
Ugh, this interacts poorly with `new-build`, now it thinks that the package has no component named `package-tests`.